### ISSUE TITLE
Separated network hassio from tuya devices.

### ISSUE THREAD

**Issue by GMTZMU** [rospogrigio/localtuya#216]
Hi, I am working with two different networks, one to Hassio Server in 192.168.1.64 and other to all Tuya devices 192.168.10.x/24.

I open all ports from 192.168.1.64 to  192.168.10.x/24 and open all ports from 192.168.10.x/24 to 192.168.1.64.

When I configure new devices to "Local Tuya" in my Hassio server, the Hassio Server can't find any devices. I configured manually, but de new device don't work correctly.

Do you have any idea, what I can configure all devices?.

Thanks.

**Comment by postlund** [comment]
You need to forward UDP broadcasts to port 6668 for it to work.

**Comment by wlatic** [comment]
What devices talks to both of these networks? Do you have any Vlans involved?

**Comment by GMTZMU** [comment]
Thanks everybody for your answers.

Yes, I have two vlans, one for my home devices (PC's, Tablets, Phones, TV's and HASSIO server), and other vlan only for domotic devices.

Two vlans are separated with a firewall and only open the necessary protocols and ports. In this case to configure Local Tuya Integration I open all protocols and ports, but only between Hassio Server and Tuya Device.

**Comment by GMTZMU** [comment]
Hello everebody.

Only is necessary configure this role in the firewall from IP hassio TCP all ports to IP Device Tuya port 6668 TCP.

And configure manually any device.

And this configuration is working correctly.

Thanks.

**Comment by paulearley** [comment]
I too am trying to get this going on a UDM Pro network appliance and VLANS. Opening up ports in the firewall does not seem to work. I ran this multicast relay (https://github.com/scyto/multicast-relay) which works great for Sonos but dues not allow localtuya discovery across 2 VLANS. This relay software allows one to specify multicast IP networks and ports. I know to forward 6668 but which multicast IP should I use in the local control set? I tried 224.0.0.1 and 224.0.0.253 to no avial.
Thanks
